### ISSUE TITLE
patch elf error and socket error

### ISSUE THREAD

**Issue by equisde** [bdloser404/Fluttermux#4]
I'm getting the following errors
![Screenshot_20231005-002659_Termux](https://github.com/bdloser404/Fluttermux/assets/34652149/850e7fe0-06f7-4496-b86c-630c1e5e81b0)

**Comment by bdloser404** [comment]
make sure your device is aarch64.

`uname -om`

expected: Android aarch64

**Comment by gandergz37** [comment]
Sabe erros. My device isso aarch64

**Comment by gandergz37** [comment]
Same error. My device is aarch64. Seems to refuse download from pub.dev. There is any way to do that by hand?

**Comment by mdHABIBURRAHMANHABIB1** [comment]
> Same error. My device is aarch64. Seems to refuse download from pub.dev. There is any way to do that by hand?

https://github.com/mumumusuc/termux-flutter 

check this out

**Comment by equisde** [comment]
yep my arch is aarch64 i know it becaus i'm building m own kernel
